test/parallel/test-net-persistent-ref-unref.js

```diff
@@ -0,0 +1,39 @@
+var common = require('../common');
+var assert = require('assert');
+var net = require('net');
+var TCPWrap = process.binding('tcp_wrap').TCP;
+
+var echoServer = net.createServer(function(conn) {
+  conn.end();
+});
+
+var ref = TCPWrap.prototype.ref;
+var unref = TCPWrap.prototype.unref;
+
+var refed = false;
+var unrefed = false;
+
+TCPWrap.prototype.ref = function() {
+  ref.call(this);
+  refed = true;
+};
+
+TCPWrap.prototype.unref = function() {
+  unref.call(this);
+  unrefed = true;
+};
```

Comment:
Can be made more rigorous if you increment/decrement a refCount variable and check that it has the expected value inside the method call (0 for ref, 1 for unref.)